test/integration/driver.go

```diff
@@ -48,8 +48,8 @@ const (
 	// budget is the maximum number of retries with 1s delays
 	budget = 90
 
-	// Mixer SHA *update manually*
-	mixerSHA = "ea3a8d3e2feb9f06256f92cda5194cc1ea6b599e"
+	// Mixer tag *update manually*
+	mixerTag = "demo"
```

Comment:
It's better to keep demo. The advantage of a dynamic tag like "demo" is we can move the tag to a newer image without making changes in the code and building a new set of images. A tag like "demo" is supposed to move only after successful verification of integration tests.  If something suddenly starts to fail, it will not be because of the mixer image which is fixed, but because of changes in manager. SHA will not help here. If you must get the SHA, you can find the image marked as demo in the container registry, and get the date/build#/sha.